### DISCUSSION TITLE
米家智能门锁状态无法获取

### DISCUSSION THREAD

**Discussion by fay000fay** [XiaoMi/ha_xiaomi_home#389] [Q&A]
设备为：lumi.lock.mcn01；该sensor.lumi_cn_1010907215_mcn01_status_p_5_1实体属性可能的状态为：Open, Close, Error；但是一直是处于“未知”状态，不管是开门还是关门。

**Comment by zraxx** [comment]
第一代智能门锁，也一样无法获取状态![Screenshot_2024-12-24-00-07-54-234_io.homeassistant.companion.android.png](https://github.com/user-attachments/assets/ee368589-9b1b-404f-9c03-b7ea488adfd0)

**Comment by zghnwsq** [comment]
未知状态可能是HA无法解析它的值，可以在开发者工具的状态中查看，通过辅助元素手动解析一下

**Reply by fay000fay** [reply]
> 未知状态可能是HA无法解析它的值，可以在开发者工具的状态中查看，通过辅助元素手动解析一下

![image](https://github.com/user-attachments/assets/b6f2f145-8d83-4435-9780-f68a9a7b4411)
开发者状态就是unknown。辅助元素怎么手动解析呢？

**Reply by zghnwsq** [reply]
参考 https://github.com/XiaoMi/ha_xiaomi_home/issues/112#issuecomment-2567245939

你这情况好像不一样，看一下调试日志吧，看看这个p_5_1属性，服务器原始返回的是啥
